app-backend/project/plugins.sbt

```diff
@@ -1,8 +1,6 @@
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.3")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.2")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.2")
-
-addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.1")
```

Comment:
There are still a handful of Bintray related volume mounts in the Docker Compose configuration files. Seems like those could be removed as well, but I have not tested it yet.